R/verb-arrange.R

```diff
@@ -1,3 +1,28 @@
+#' Arrange rows by variables in a local lazy tibble
```

Comment:
I _think_ it's better to say "remote database table" rather than "local lazy tibble" (just because that's exposing internal implementation details which users shouldn't have to care about)

Reply:
I had a feeling this would be the case. It's fixed [here](https://github.com/tidyverse/dbplyr/pull/222/commits/6873d6482a1cfa076b4d49d0787485ab996cc3cd#diff-12fbbf8bf4196bb24a276dff57ec3116R1)